seed/migrations/0123_rule_condition.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.10 on 2020-03-06 19:19
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('seed', '0122_remove_rule_condition'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='rule',
+            name='condition',
```

Comment:
It looks like there are three migrations to add, remove, then add the condition again. Is that required? Seems like 0119 through 0123 can be squashed?